javascript-data-model/order-history.js

```diff
@@ -0,0 +1,55 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    orderDate: 'August 4, 2020',
+    totalCost: '$34.00',
+    title: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    itemCost: '$31.55',
+    deliveredDate: 'August 8, 2020',
+    returnWindowClosed: 'September 7, 2020'
+  },
```

Comment:
In this object, data on the order is intermixed with data about the specific item ordered. This can be confusing and is unintuitive for immediately understanding the data. You should instead have data on the order itself in this object, then in this object, have a property that stores data on the item(s) ordered.